app/models/form.rb

```diff
@@ -5,20 +5,38 @@ class Form
 
   include Form::Setup
 
-  def initialize(form_path, name)
-    raise "No form definition file exists for given year".freeze unless File.exist?(form_path)
-
-    @name = name
-    @setup_sections = [Form::Setup::Sections::Setup.new(nil, nil, self)]
-    @form_definition = JSON.parse(File.open(form_path).read)
-    @form_sections = form_definition["sections"].map { |id, s| Form::Section.new(id, s, self) }
-    @type = form_definition["form_type"]
-    @sections =  setup_sections + form_sections
-    @subsections = sections.flat_map(&:subsections)
-    @pages = subsections.flat_map(&:pages)
-    @questions = pages.flat_map(&:questions)
-    @start_date = Time.iso8601(form_definition["start_date"])
-    @end_date = Time.iso8601(form_definition["end_date"])
+  def initialize(form_path, name, sections_in_form = [], type = "lettings")
+    if type == "sales"
+      # @form_definition = sections.map do |s|
+      #   s.new(nil, nil, self)
+      # end
+      @name = name
+      # @setup_sections = [Form::Setup::Sections::Setup.new(nil, nil, self)]
```

Comment:
Are these comments still needed?

Reply:
I think they'll be needed in the near future - currently sales form only has one section and one question but will have more as we work on the remaining tickets. Happy to remove this in the meantime though 🙂

Reply:
Don't they all get passed into as params though? We don't need to separate this from line 16 for Sales do we?

Reply:
Have a look now? Changed how we're creating the form to be a bit more like the lettings form - separated the setup section https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/858/commits/0ef59ce0bf35143113471d732133975d57c5e7ab